apps/chrome-devtools/project.json

```diff
@@ -164,7 +164,7 @@
         "{projectRoot}/dist/options.html"
       ],
       "inputs": [
-        "global",
+        "base",
```

Comment:
this seems to be too many inputs for `copy-assets` tasks (`${workspaceRoot}/tsconfig.base.json` for example shouldn't invalidate this cache)

Reply:
shouldn't we have an asset input?

Reply:
`${workspaceRoot}/tsconfig.base.json` is actually the only from the list that may not be part of it.

Then there is 3 choices:
- keep it in
- create a new `nameInputs` for assets (duplicate of `base` without this file + `{projectRoot}/src/assets/**` used only here.
- put the content of `base` flatten here

I see more advance to keeping it actually

Reply:
or option , adding `!${workspaceRoot}/tsconfig.base.json` :D
I'm ok to keep if it's the only one

Reply:
`!${workspaceRoot}/tsconfig.base.json` in this file is also a solution indeed, I go for it